api/metadata_handler.go

```diff
@@ -112,7 +200,7 @@ func (h MetadataHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-// filterMetadata filters the given metadata down to entries where the value (links) contain
+// filterMetadata filters the given Metadata down to entries where the value (links) contain
```

Comment:
Why capitalize this? 'metadata' here seems like a reference to the input parameter name